src/build/smart-contracts.md

```diff
@@ -1,153 +1,133 @@
-
-- **increaseAllowance**
-
-Increase the allowance limit of spender for a given caller
-
-- **decreaseAllowance**
-
-Decrease the allowance limit of spender for a given caller
-
-- **transferFrom**
-
-Used to transfer token from src to target and caller must be the address who is pre approved by the src. Pre approve of address is done by increaseAllowance
-
-### Queries
-
-- **balance**
-
-Display balance of given address
-
-- **allowance**
-
-Display allowance limit spender has for a given address
+```
+rustup --version
+rustup default stable
+default stable
+cargo --version
+# If this is lower than 1.51.0+, update
+rustup target list --installed
+```
 
-- **allAllowances**
+Update all dependencies in the lockfile for cargo and rustup:
+```
+cargo update
+rustup update
+```
 
-Display list of all address who are allowed to spend from given owner address
+## Deployment and interaction of CW20
 
-- **allAccounts**
+[CW20](https://github.com/CosmWasm/cw-plus/tree/v0.9.0/contracts/cw20-base) is equivalent to ERC20. CW-20 is similar, in some respects, to bitcoin, Litecoin, and any other cryptocurrency. CW-20 tokens are blockchain-based assets that have value and can be sent and received. The primary difference is that instead of running on their own blockchain, CW-20 tokens are issued on the CosmWasm network.
 
-Display paginated list of all accounts with balance \&gt; 0
+This section explains how to deploy and interact with CW20 using cudos-noded.
 
-- **tokenInfo**
+### Functions present in CW20 base standard
 
-Display name and symbol of a token
+The specification of CW20 are represented within the following:
+* [Base](https://github.com/CosmWasm/cw-plus/blob/main/packages/cw20/README.md#base)
+* [Allowances](https://github.com/CosmWasm/cw-plus/blob/main/packages/cw20/README.md#allowances)
+* [Mintable](https://github.com/CosmWasm/cw-plus/blob/main/packages/cw20/README.md#mintable)
 
-- **minter**
-Display a minter address for a given contract
+Note that you can find all messages, actions, and queries within the list above.
 
-## Download Node and Code
+### Get the binaries and download the CosmWasm Plus contracts
 
-- Setup a CUDOS\_NODED using binary builder mentioned here [https://github.com/CudoVentures/cudos-network-docs/blob/Table-of-Content-folders/src/docs/build-and-earn/testnet-guides/start-binaries.md](https://github.com/CudoVentures/cudos-network-docs/blob/Table-of-Content-folders/src/docs/build-and-earn/testnet-guides/start-binaries.md)
-- Download [cosmwasm-plus](https://github.com/CosmWasm/cw-plus) contract and compile the contracts using docker(make sure you run docker command inside the root folder)
+* Follow the [instructions to start and build the binaries](/build/start-binaries)
+* Clone the [cw-plus](https://github.com/CosmWasm/cw-plus) repo with [release tag of v0.9.1](https://github.com/CosmWasm/cw-plus/tree/v0.9.0) by running the command:
+```
+git clone --depth 1 --branch v0.9.1 https://github.com/CosmWasm/cw-plus.git
```

Comment:
@UrAvgDeveloper you had a different view on the version that should be used?

Reply:
as long as we are using stable version of cosmwasm-std version 0.16.0 we don't have to worry.